packages/core/src/orchestrate/call.ts

```diff
@@ -31,7 +33,10 @@ import { StreamUtil } from "../utils/StreamUtil";
 
 import { cancelFunction } from "./internal/cancelFunction";
 
-export async function call<Model extends ILlmSchema.Model>(ctx: AgenticaContext<Model>): Promise<AgenticaPrompt<Model>[]> {
+export async function call<Model extends ILlmSchema.Model>(
+  ctx: AgenticaContext<Model> | MicroAgenticaContext<Model>,
+  operations: AgenticaOperation<Model>[],
+): Promise<AgenticaPrompt<Model>[]> {
```

Comment:
Return type is ambigous

we can use `Promise<(AgenticaCancelPrompt<Model> | AgenticaExecutePrompt<...> | AgenticaTextPrompt<...>)[]>`

Reply:
The micro agentica can't receive cancel prompt

Reply:
Return type should be changed (and specialized exactly)